src/registrar/admin.py

```diff
@@ -2864,34 +2863,204 @@ def save_model(self, request, obj, form, change):
+        """Returns a list of links for each related domain request"""
+        queryset = obj.get_domain_requests()
+        return self.get_field_links_as_list(queryset, "domainrequest", link_info_attribute="get_status_display")
+
+    domain_requests.short_description = "Domain requests"  # type: ignore
+
     # Creates select2 fields (with search bars)
     autocomplete_fields = [
         "creator",
         "federal_agency",
         "senior_official",
     ]
 
-    def change_view(self, request, object_id, form_url="", extra_context=None):
-        """Add related suborganizations and domain groups"""
-        obj = self.get_object(request, object_id)
+    def get_field_links_as_list(
+        self, queryset, model_name, attribute_name=None, link_info_attribute=None, seperator=None
+    ):
+        """
+        Generate HTML links for items in a queryset, using a specified attribute for link text.
+
+        Args:
+            queryset: The queryset of items to generate links for.
+            model_name: The model name used to construct the admin change URL.
+            attribute_name: The attribute or method name to use for link text. If None, the item itself is used.
+            link_info_attribute: Appends f"({value_of_attribute})" to the end of the link.
+            separator: The separator to use between links in the resulting HTML.
+            If none, an unordered list is returned.
 
-        # ---- Domain Groups
-        domain_groups = DomainGroup.objects.filter(portfolio=obj)
+        Returns:
+            A formatted HTML string with links to the admin change pages for each item.
+        """
+        links = []
+        for item in queryset:
 
-        # ---- Suborganizations
-        suborganizations = Suborganization.objects.filter(portfolio=obj)
+            # This allows you to pass in attribute_name="get_full_name" for instance.
+            if attribute_name:
+                item_display_value = self.value_of_attribute(item, attribute_name)
+            else:
+                item_display_value = item
 
-        extra_context = {"domain_groups": domain_groups, "suborganizations": suborganizations}
+            if item_display_value:
+                change_url = reverse(f"admin:registrar_{model_name}_change", args=[item.pk])
+
+                link = f'<a href="{change_url}">{escape(item_display_value)}</a>'
+                if link_info_attribute:
+                    link += f" ({self.value_of_attribute(item, link_info_attribute)})"
+
+                if seperator:
+                    links.append(link)
+                else:
+                    links.append(f"<li>{link}</li>")
+
+        # If no seperator is specified, just return an unordered list.
+        if seperator:
+            return format_html(seperator.join(links)) if links else "-"
+        else:
+            links = "".join(links)
+            return format_html(f'<ul class="unstyled-list-elements">{links}</ul>') if links else "-"
```

Comment:
Love it, but look into `add-list-reset`. You might be able to avoid creating a new class: https://designsystem.digital.gov/utilities/list-reset/